JsonSchema.Tests/GithubTests.cs

```diff
@@ -875,4 +875,25 @@ public void Issue506_ReffingIntoAdditionalProperties()
 
 		Assert.IsNotNull(subSchema);
 	}
+
+	[TestCase(@"{""additionalItems"":""not-a-schema""}", 0, 33)]
+	[TestCase(@"{""additionalProperties"":""not-a-schema""}", 0, 38)]
+	[TestCase(@"{""allOf"":[""not-a-schema""]}", 0, 24)]
+	[TestCase(@"{""anyOf"":[""not-a-schema""]}", 0, 24)]
+	[TestCase(@"{
+  ""$schema"": ""https://json-schema.org/draft/2020-12/schema"",
+  ""required"": [ { ""abc"": null } ]
+  ""properties"": {
+      ""abc"": { ""$ref"": ""mySchema.json"" }
+  },
+  ""additionalProperties"": false
+}", 2, 17)]
+	public void Issue517_IncorrectJsonExceptionpointer(string schemaStr, long expectedLineNumber, long expectedBytePositionInLine)
```

Comment:
Can you rename this, please? "Pointer" suggests JSON Pointer.

Also, are the values in the exception `long`s?  The expected values' types should match.

Reply:
I gave `long` because that's what's in `JsonException`, but all of the examples I'm writing are going to be short enough that it won't need a `long`. I could go either way with the type for the test.

Reply:
I didn't have time to check what was in the exception at the time.  Let's use what's there.  (Though I'm surprised they used a long.  Do they expect lines longer than 2 billion chars?)